app/javascript/packages/document-capture/components/acuant-capture.tsx

```diff
@@ -615,6 +645,34 @@ function AcuantCapture(
           )}
         </AcuantCamera>
       )}
+      {isCapturingEnvironment && selfieCapture && (
+        <AcuantSelfieCamera
+          onImageCaptureSuccess={onSelfieCaptureSuccess}
+          onImageCaptureFailure={onSelfieCaptureFailure}
+          onImageCaptureOpen={() => setIsCapturingEnvironment(true)}
+          onImageCaptureClose={() => setIsCapturingEnvironment(false)}
+        >
+          {/* TODO Bug (hooks related)
```

Comment:
@aduth If you have a minute, I'd appreciate extra 👀 on this. It seems like a state bug having to do with the acuant permission failure cookie, but I'm having a hard time tracking it down.

Reply:
If I'm following correctly, this isn't so much a bug as much as the browser remembering the declined preference and automatically declining it from that point forward. I recall us having spent some time investigating this in the past, and how the only way around this is for the user to manually change their browser settings to remove the camera preference.

Reply:
Some related tickets:

- https://cm-jira.usa.gov/browse/LG-4390
- https://cm-jira.usa.gov/browse/LG-4671

Reply:
Thanks for those links! They reinforce what we talked about in standup yesterday: I'm going to write this up as something to research in a separate ticket so it doesn't block this work.